### DISCUSSION TITLE
Would be great to configure api_url before worker starts

### DISCUSSION THREAD

**Discussion by rantiev** [mswjs/msw#1235] [Q&A] [answered]
### Scope

Adds a new behavior

### Compatibility

- [ ] This is a breaking change

### Feature description

In order to avoid regexps in every API mock and API_URL string usage in every API mock it would be great to have an option to configure alias for API path, instead of localhost, many servers are located in different domains than FE part of application.

**Comment by kettanaito** [comment, marked as answer]
Hey, @rantiev. 

You can use a [Base URL with your handlers](https://www.youtube.com/watch?v=N7kvgJvxcsw) at any time. This doesn't need any additional configuration on the library's side: repetitive URLs is a part of your logic, so it's your logic that can abstract them. Let me know if this suggestion suits your use case. If not, I'd like to hear more details and some code examples related to what you're trying to do.

**Reply by kettanaito** [reply]
I'm marking this as the answer because I think your question is about using base URLs. Please comment underneath if I got it wrong, we can discuss and find the solution together.
